README.md

```diff
@@ -15,6 +15,75 @@ If you can't find what you are looking for, try the [old docs on the ODK website
 
 The following steps help you build and view ODK docs locally. For details on each of these steps, see the [Contributor Guide](http://docs.opendatakit.org/contributing).
 
+### Prerequisites
+
+ * Install Docker
+   * Windows and Mac users should follow the instructions at [the get started guide](https://www.docker.com/get-started)
+   * Linux users should follow the instructions for their specific distribution: [CentOS](https://docs.docker.com/install/linux/docker-ce/centos/), [Debian](https://docs.docker.com/install/linux/docker-ce/debian/), [Fedora](https://docs.docker.com/install/linux/docker-ce/fedora/), [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/), [Binaries](https://docs.docker.com/install/linux/docker-ce/binaries/)
+   
+   More info at the [Docker CE docs page](https://docs.docker.com/install/)
+ * Install [Git-LFS](https://git-lfs.github.com/).
+
+### Cloning the repo
+
+After that, you need to clone the docs repo
+
+```
+git clone https://github.com/opendatakit/docs.git
+```
+
+It can take a long time (>10 minutes) to clone the repo due to the large number of images in the docs. If you get an error such as `Smudge error` or `github's rate limit reached`, run `git checkout -f HEAD` until you get the message `Checking out files: 100% done`.
+
+### Building the Docker image
+
+Next, you need to build the Docker image with all the tools you will be using to work with ODK's docs.
+
+```
+docker build -t odk-docs .
+```
+
+It can take a long time to build the Docker image, but you only need to do this once.
+
+**Windows users**
+ * All commands should be run in an elevated PowerShell window. Right click on it and select the "Run as administrator" option.
+ * Ensure Docker is running by checking your system tray. If Docker is not running, launch "Docker for Windows" app and wait until a notification confirms that Docker is running.
+
+### Building and serving the docs locally 
+
+Build and serve the docs locally with:
+ * Windows: `.\run-task.bat odk1-autobuild`
+ * Linux/macOS: `./run-task.sh odk1-autobuild`
+ 
+You can then view the docs in your browser at http://localhost:8000.
```

Comment:
s/8000/8080

Because the previous can take a long time, maybe we can say, something like. Once your terminal shows a "Serving on http://0.0.0.0:8080" message, you can view the http://localhost...

I'd also add "Changes you make in the source files will automatically be built and shown in your browser"